generators/app/index.js

```diff
@@ -9,27 +9,35 @@ var semver = require('semver');
 var versions = require('./dependency-versions.js');
 var yosay = require('yosay');
 var _ = require('lodash');
+var genConfig = null;
 
 module.exports = yeoman.Base.extend({
+  _getConfigValue: function(key) {
+    var configValue = this.config.get(key);
+    var defaultValue = genConfig[key];
+    if(null != configValue) {
```

Comment:
In idiomatic JavaScript we'd probably write this as ```if(configValue)```. Javascript has a really weird sense of truthyness and falsyness... So if ```configValue``` may have the value ```false```, then we have to be trickier. In this case I think we should use ```if(undefined !== configValue)```. Unless of course ```config.get()``` returns null instead of undefined to indicate there is no such config (that would be a bad API choice for whoever wrote the config class) ... if ```config.get()``` returns ```null``` when the requested property doesn't exist then I would re-write this as ```if(null !== configValue)```. (See comment on line 181 for rational.)